### PR TITLE
Update active nav item

### PR DESCRIPTION
**Pull request by davidpmccormick**
Correctly highlight the site section we're in

screen shot 2018-03-02 at 13 14 11

### CODE REVIEW

File: catalogue/webapp/pages/work.js [hunk 1]
```diff
@@ -180,6 +180,7 @@ const WorkPage = ({ work }: Props) => {
       type={'website'}
       url={`https://wellcomecollection.org/works/${work.id}`}
       imageUrl={iiifImage({size: '800,'})}
+      siteSection='images'
```
Comment (jamesgorrie) [inline catalogue/webapp/pages/work.js]: Is this really the right name?
Could we call it catalogue?

File: catalogue/webapp/pages/work.js [hunk 2]
```diff
@@ -209,8 +210,7 @@ const WorkPage = ({ work }: Props) => {
                 ])}>
                   <Icon name='underConstruction' extraClasses='margin-right-s2' />
                   <p className={`${font({s: 'HNL5', m: 'HNL4'})} no-margin`}>
-                    We’re improving the information on this page.
-                    <a href='/progress'>Find out more</a>.
+                    We’re improving the information on this page. <a href='/progress'>Find out more</a>.
```
Comment (jamesgorrie) [inline catalogue/webapp/pages/work.js]: Is the space compressed in next?

File: common/views/components/DefaultPageLayout/DefaultPageLayout.js [hunk 3]
```diff
@@ -62,7 +62,7 @@ export const TwitterCard = ({
   type,
   url,
   title,
-  description,
+  description = '',
```
Comment (jamesgorrie) [inline common/views/components/DefaultPageLayout/DefaultPageLayout.js]: Could we get this?

File: catalogue/webapp/pages/work.js (hunk 1 above)
Comment (davidpmccormick) [reply catalogue/webapp/pages/work.js]: Could we update it at the point we update the nav label? If not, we'll have three independent words for the same section!

Comment (jamesgorrie) [reply catalogue/webapp/pages/work.js]: 💯

File: catalogue/webapp/pages/work.js (hunk 2 above)
Comment (davidpmccormick) [reply catalogue/webapp/pages/work.js]: yeah

File: common/views/components/DefaultPageLayout/DefaultPageLayout.js (hunk 3 above)
Comment (davidpmccormick) [reply common/views/components/DefaultPageLayout/DefaultPageLayout.js]: We get it when it exists, but when it doesn't we're still passing it to `striptags()` (which tries to do a string replace). I suppose the better solution is to only conditionally pass it to striptags.

**Pull request merged**